packages/xo-web/src/xo-app/host/tab-advanced.js

```diff
@@ -112,10 +113,13 @@ export default class extends Component {
       return uniqPacks
     }
   )
+  _setIscsiHost = iscsiIqn =>
+    editHost(this.props.host, { iscsiIqn: iscsiIqn === '' ? null : iscsiIqn })
```

Comment:
Should be able to delete the `iscsiIqn` ?

Reply:
No, it's required.